### DISCUSSION TITLE
On formatting

### DISCUSSION THREAD

**Discussion by nabw** [nextflow-io/nextflow#2301] [Q&A] [answered]
Hi everyone, I was trying to create a MWE to file a bug where my generated code has some deleted portions, but for some reason it fails to work. I guess it is a dumb mistake, so I'm posting it as a general question. This is the script:

```
#!/usr/bin/env nextflow
nextflow.enable.dsl=2

process printfiles {

    output:
        path '*.out', emit: asdf
        
    """
    #!/bin/bash
    python -c \"print(2)\" > TEMP
    grep '2' TEMP | sed -E 's/.* ([0-9]+\.[0-9]+)s.*/\1/g' > TIMES
    """
}

workflow WORK {
    printfiles()
}
```

The 'sed' part is the one that eats code in my actual code, but I can't test it here as it fails to compile:

- cause: Unexpected input: '{' @ line 6, column 20.
   process printfiles {

Thanks in advance.

**Comment by bentsherman** [comment, marked as answer]
Hi @nabw , answering your question since no one answered. I think the issue is that you need to escape backslash characters in your task script, so every `\` should be `\\`.

The error message you got is vague because it only points to the process in which the error occurred. Unfortunately this kind of error is common, but it usually means there is a syntax error within the process or config block.

**Reply by nabw** [reply]
Better late than never, as I actually never got that fixed. Thank you!
